### PR TITLE
Update kustomization yaml to add integration tests for new application

### PR DESCRIPTION
**Pull request by savitaashture**
Update Integration Test Scenario for Enterprise Contract Validation for tekton-ecosystem workspace and ecosystem-images application

### CODE REVIEW

**Comment by savitaashture** [comment]
@scoheb @gbenhaim @ernesgonzalez33 Please review and merge the PR
Thank you

**Review by ernesgonzalez33 — CHANGES_REQUESTED** [review]
@savitaashture do you have an RPA already? If that's the case, what policy are you using?

**Comment by savitaashture** [comment]
> @savitaashture do you have an RPA already? If that's the case, what policy are you using?

sorry what is RPA ?

File: auto-generated/cluster/stone-prd-rh01/tenants/tekton-ecosystem-tenant/appstudio.redhat.com_v1beta1_integrationtestscenario_ecosystem-images-enterprise-contract.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+apiVersion: appstudio.redhat.com/v1beta1
+kind: IntegrationTestScenario
+metadata:
+  name: ecosystem-images-enterprise-contract
+  namespace: tekton-ecosystem-tenant
+spec:
+  application: ecosystem-images
```
Comment (savitaashture) [inline auto-generated/cluster/stone-prd-rh01/tenants/tekton-ecosystem-tenant/appstudio.redhat.com_v1beta1_integrationtestscenario_ecosystem-images-enterprise-contract.yaml:7]: @ernesgonzalez33 I have a query here
under one application we can have multiple component so the question is does integration tests are same for all component under one application
basically ask is integration tests are across application ?

Comment (ernesgonzalez33) [reply auto-generated/cluster/stone-prd-rh01/tenants/tekton-ecosystem-tenant/appstudio.redhat.com_v1beta1_integrationtestscenario_ecosystem-images-enterprise-contract.yaml:7]: Yes. Integration tests cover the entire application

**Comment by ernesgonzalez33** [comment]
> > @savitaashture do you have an RPA already? If that's the case, what policy are you using?
> 
> sorry what is RPA ?

This is part of the release creation. Don't worry about it if you haven't reached that part

**Review by ernesgonzalez33 — APPROVED** [review]

**Pull request merged**